drivers/rtc/rtc_ll_stm32.c

```diff
@@ -67,10 +67,21 @@ LOG_MODULE_REGISTER(rtc_stm32, CONFIG_RTC_LOG_LEVEL);
 /* Timeout in microseconds used to wait for flags */
 #define RTC_TIMEOUT 1000000
 
+/* RTC calibration output frequency representations
+ * COSEL: Calibration output selection
+ * 0: Calibration output is 512 Hz
+ * 1: Calibration output is 1 Hz
+ */
+#define RTC_CALIB_OUT_FREQ_VAL_1   (1)
+#define RTC_CALIB_OUT_FREQ_VAL_512 (0)
```

Comment:
Note that you can directly use defines provided by LL API:
```
  *         @arg @ref LL_RTC_CALIB_OUTPUT_1HZ
  *         @arg @ref LL_RTC_CALIB_OUTPUT_512HZ
```

Reply:
good idea, i did update